### PR TITLE
Number of links appear on homepage when hover

### LINKED ISSUE #191: Number of links appear on hover on homepage itself
The number of links added now can be seen only when someone gets into the profile, that number can be made more useful to one when it appears on the homepage when hovered over the profile logo, along with the user name.
image

image

### PR DESCRIPTION
**Pull request by nathgoutam93**
## Fixes Issue

Closes #191 

## Change List

- [X] added number of links badge on users component
- [X] removed the number of links badge from the profile component

## Check List 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

https://user-images.githubusercontent.com/91387097/146555794-46c47df8-5c5f-4e23-8c00-57d75a7ef200.mp4

chrome_pCyHya4Pfk

on small screen sizes it shows by default

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/824"></a>

### CODE REVIEW

**Comment by krishguptadev** [comment]
Hmm, @Panquesito7 This is a feature and should be `aspect: code`

**Comment by Panquesito7** [comment]
> Hmm, @Panquesito7 This is a feature and should be `aspect: code`

I don't think so. It makes a change that adds a *text*, it changes *text*.

**Comment by krishguptadev** [comment]
@nhcarrigan what do you think?

**Comment by nhcarrigan** [comment]
Correct - `aspect: text` is for documentation :)

File: src/Components/Home/Home.css [hunk 1]
```diff
@@ -11,12 +11,32 @@ a {
 
 .p-chip:hover {
   transform: scale(1.1);
-  box-shadow: rgba(255, 255, 255, 0.2) 0px 0px 0px 1px inset,
-    rgba(0, 0, 0, 0.9) 0px 0px 0px 1px;
+  background: linear-gradient(145deg, #c8cbcf, #eef2f6);
+  box-shadow: 8px 8px 15px #bdc0c4, -8px -8px 15px #ffffff;
```
Comment (emmalearnscode) [inline src/Components/Home/Home.css:15]: Why did you change the previous box-shadow?

File: src/Components/Home/Home.css [hunk 2]
```diff
@@ -11,12 +11,32 @@ a {
...
+}
+
+.p-chip .p-avatar img {
```
Comment (emmalearnscode) [inline src/Components/Home/Home.css:18]: Is this custom CSS not something that can be achieved with prime react classes? (lines 18 - 30)

File: src/Components/Home/Users.js [hunk 3]
```diff
@@ -37,18 +39,35 @@ function Users({ list }) {
...
+                    <Avatar
+                      image={user.avatar}
+                      size="large"
+                      className="p-overlay-badge"
+                      onImageError={(error) => {
+                        utils.setDefaultSVG(user.name, error)
+                      }}
+                    >
+                      <Badge
+                        value={user.linkCount > 3 ? '3+' : user.linkCount}
```
Comment (emmalearnscode) [inline src/Components/Home/Users.js:54]: Why 3? I would suggest 9 as double digits could possibly affect styling but 3 seems very arbitrary.

Comment (krishguptadev) [reply src/Components/Home/Users.js:54]: +1, Even I have more than 2 links

Comment (krishguptadev) [inline src/Components/Home/Users.js:54]:
```suggestion
                        value={user.linkCount > 9 ? '9+' : user.linkCount}
```

File: src/Components/Home/Home.css (hunk 2 above)
Comment (nathgoutam93) [reply src/Components/Home/Home.css:18]: prime-react is an UI component library it has no small CSS classes to work with in this case, the prime-flex has some but those are related to layout. Please  correct me if I'm wrong , have not prior experience using prime-react.

File: src/Components/Home/Home.css (hunk 1 above)
Comment (emmalearnscode) [reply src/Components/Home/Home.css:15]: Ok 👍🏼

File: src/Components/Home/Home.css (hunk 2 above)
Comment (emmalearnscode) [reply src/Components/Home/Home.css:18]: Just wanted to check, I’m not well-versed in prime react either, I just know we’re trying to avoid custom css as much as possible.

**Review by emmalearnscode — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by schmelto — APPROVED** [review]
